code/src/layouts/dashboard-tabs/DashboardTabs.scss

```diff
@@ -0,0 +1,44 @@
+@import "@design-system";
+
+.dashboard-tabs {
+  display: flex;
+  gap: spacing(8);
+
+  &__label-container {
+    display: flex;
+    flex-shrink: 0;
+    flex-basis: 270px;
+    flex-direction: column;
+    gap: spacing(2);
+  }
+
+  &__label-item {
+    @include transition((background-color));
+    display: flex;
+    align-items: center;
+    gap: spacing(3);
+    padding: spacing(4) 0 spacing(4) spacing(4);
+    cursor: pointer;
+    border-left: 3px solid $transparent;
+
+    &:hover {
+      background-color: var(
+        --spa-dashboard-tabs-hover-background-color,
+        $ghost-alpha
+      );
+    }
+
+    &--active {
+      border-color: var(--spa-dashboard-tabs-active-color, $purple);
+      background-color: var(
+        --spa-dashboard-tabs-active-bg-color,
```

Comment:
`bg` full name as you made above

<img width="326" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/f51e9361-1d21-4fb7-83c9-a05511699569">

Reply:
Yeap, fixed